ack/readers/twitter/reader.py

```diff
@@ -316,9 +316,22 @@ def get_cards_report(self):
         Supported entities: CARD
         Documentation: https://developer.twitter.com/en/docs/ads/creatives/api-reference/
         """
+        tweets_with_uris = []
         for tweet in self.get_published_tweets_generator():
             if "card_uri" in tweet:
-                card_fetch = self.get_card_fetch(card_uri=tweet["card_uri"])
+                tweets_with_uris.append(tweet)
+
+        batch_tweets = [tweets_with_uris[i:i + 200] for i in range(0, len(tweets_with_uris), 200)]
```

Comment:
What is this "200"? An arbitrary batch size? The max size allowed by the Twitter API?
It would be better to put it in a constant parameter

Reply:
it is the maximal batch size accepted by twitter api.